google/cloud/dialogflow_cx/README.md

```diff
@@ -1,9 +1,13 @@
-# Dialogflow API C++ Client Library
+# Dialogflow CX API C++ Client Library
 
 :construction:
 
 This directory contains an idiomatic C++ client library for the
-[Dialogflow API][cloud-service-docs], a service to Builds conversational interfaces (for example, chatbots, and voice-powered apps and devices).
+[Dialogflow API][cloud-service-docs], a service to build conversational
+interfaces (for example, chatbots, and voice-powered apps and devices).
+Dialogflow supports two [editions], this library supports the CX edition.
```

Comment:
I would say: "There are [two editions] of Dialogflow. This library supports the CX edition."

Reply:
Done.